client/finality-grandpa/src/communication/mod.rs

```diff
@@ -690,7 +677,7 @@ pub(crate) fn check_message_sig_with_buffer<Block: BlockT>(
 /// use the same raw message and key to sign. This is currently true for
 /// `ed25519` and `BLS` signatures (which we might use in the future), care must
 /// be taken when switching to different key types.
-struct OutgoingMessages<Block: BlockT> {
+pub struct OutgoingMessages<Block: BlockT> {
```

Comment:
```suggestion
pub(crate) struct OutgoingMessages<Block: BlockT> {
```
That would be enough, right?